### DISCUSSION TITLE
Welcome to ioBroker.go-e Discussions!

### DISCUSSION THREAD

**Discussion by MK-2001** [MK-2001/ioBroker.go-e#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DaveMe81** [comment]
Hello I got a question.
I recently installed this go-e adapter into my iobroker. In the configuration below "foreign objects" I can put in data point from other adapters like Senec - my battery. 
My question is about the value of the household. In my case the power of the connected wallbox charger will also be included in the household. Does this go-e adapter knows this? So does it caculate the own consuption out of the household value?

**Reply by MK-2001** [reply]
Hi, currently not. May be it could become a feature. :-) 

But to understand I will create an example:
you PhotoVoltaic creates 5.000 W
your housConsumtion (with the car) is at 3000W

That results in a level of 2000 W what is more possible. The adapter will now increase the Amperelevel for these 2000 W.

I do not really need to know how much the car especially is consuming. And da don't have to calculate it to the houseConsumtion or out of it.

Can you explain me why you need it?

**Reply by DaveMe81** [reply]
Thank for the fast reply.

I still startet in thinking about to optimizse the changing process. Honestly I still do not understand by 100% what these values will affect and what the adapter - so your calculation - is doing. I  was just thinking about to get the correct overall power overplus to decide what components in my household I can switch on or off. Like cooling, washing machine, etc. but also the charging process.

My ideas are simple: I want to optimize my own power consumption at all, so using as much as I can my own power generation from PV (max 9,75kWh). Before I "buy" power from the net, I want to use my own. I want to make a small dashboard to plan the charging of the eCar. My idea is to set a base level min. battery power of my eCar, let's say to do daily travel to work and shoping center. lets says 100km with idk xkW accu level. Then when I know I want to travel longer distances in let's say 2 days, I want to use the PV power first. An assumption for the next 2 days I get already. With this I can start to plan the charging over the next 2 days.

I can not imaging, that I am the first one with these ideas. And I am not that good in programming scripts. So I'm trying to find "out-of-the-box"-solutions first. The challeging is, the coordintion of different systems. If the wb would be from the same supplier like my battery and if there would be a direct connection to my house heating, then there is a solution available, but not if there are other components from other vendors. For this I need ioBroker or other to optimize my own situation.

**Reply by MK-2001** [reply]
Okay. Let me say in that way:
The Adapter tries to take the rest what is available, before just a watt will be spent to your power supplier.
